Packs/BmcHelixRemedyForce/Scripts/BMCHelixRemedyforceCreateServiceRequest/BMCHelixRemedyforceCreateServiceRequest.py

```diff
@@ -0,0 +1,348 @@
+            args.get('queue_id'),
+            args.get('queue'),
+            GET_COMMANDS["queue_id"],
+            {"queue_name": args.get('queue')},
+            using_argument
+        ),
+        'staff_id': get_field_id(
+            args.get('staff_id'),
+            args.get('staff_user_name'),
+            GET_COMMANDS["staff_id"],
+            {"username": args.get('staff_user_name'), "is_staff": True},
+            using_argument
+        ),
+        'status_id': get_field_id(
+            args.get('status_id'),
+            args.get('status'),
+            GET_COMMANDS["status_id"],
+            {"status_name": args.get('status')},
+            using_argument
+        ),
+        'urgency_id': get_field_id(
+            args.get('urgency_id'),
+            args.get('urgency'),
+            GET_COMMANDS["urgency_id"],
+            {"urgency_name": args.get('urgency')},
+            using_argument
+        ),
+        'impact_id': get_field_id(
+            args.get("impact_id"),
+            args.get("impact"),
+            GET_COMMANDS["impact_id"],
+            {"impact_name": args.get("impact")},
+            using_argument
+        ),
+        'account_id': get_field_id(
+            args.get("account_id"),
+            args.get("account"),
+            GET_COMMANDS["account_id"],
+            {"account_name": args.get("account")},
+            using_argument
+        ),
+        'service_request_definition_id': get_service_request_definition_id(
+            args.get("service_request_definition_id"),
+            args.get("service_request_definition"),
+            GET_COMMANDS["service_request_definition_id"],
+            {"service_request_definition_name": args.get("service_request_definition")},
+            using_argument
+        ),
+        'service_request_definition_params': args.get("service_request_definition_params")
+    }
+    command_args = generate_command_args_with_additional_fields(additional_fields)
+    command_args = remove_empty_elements(command_args)
+    if using_argument:
+        command_args["using"] = using_argument
+    command_res = demisto.executeCommand('bmc-remedy-service-request-create', command_args)
+    result = {}
+    try:
+        entry = command_res[0]
+        context_output_type = find_entry_type(demisto.get(entry, "Type"))
+        context_outputs = demisto.get(entry, "EntryContext")
+        human_readable = demisto.get(entry, "HumanReadable")
+        contents_format = demisto.get(entry, "ContentsFormat")
+        if isError(entry):
+            return_error(entry['Contents'])
+        else:
+            record_data = demisto.get(entry, 'Contents')
+            if not record_data:
+                return_error(ERROR_MESSAGES + json.dumps(entry))
+            else:
+                if demisto.get(record_data, "ErrorMessage"):
+                    result = record_data['ErrorMessage']
+                else:
+
+                    # Output entry
+                    result = {
```

Comment:
Use `CommandResults` https://xsoar.pan.dev/docs/integrations/context-and-outputs